### PR TITLE
Universal search default

### PR DESCRIPTION
**Pull request by somexpert**

### CODE REVIEW

**Comment by somexpert** [comment]
This will close #113. Defaults the search type to the new "Any Type", which, as you might expect, searches for the value in all name types. This also fixes an issue that existed where the search bar in the navbar would not remember your previous search type (due to it using a nonexistent variable in the template). This will also bump the version to 1.2.0.

**Comment by somexpert** [comment]
ping @ldko, @runderwood.

File: name/utils.py [hunk 1]
```diff
@@ -62,9 +62,12 @@ def resolve_type(q_type):
 
     q_type = q_type.split(',')
 
-    # Iterate through the Name Types and create a list of Name IDs
-    # using the corresponding strings values in q_type
-    return [k for k, v in Name.NAME_TYPE_CHOICES if v in q_type]
+    if "Any Type" in q_type:
+        return range(5)
```
Comment (runderwood) [inline name/utils.py]: Small quibble, but using `range(5)` here assumes the values of these constants will never change.

Would it make sense to do something like `[k for k, v in Name.NAME_TYPE_CHOICES]`?

Comment (somexpert) [reply name/utils.py]: ah, yes, that would be a much better implementation. shortsightedness due to quick dev testing on my part.

Comment (somexpert) [reply name/utils.py]: I now remember why I did that initially. I had at first actually added another value to NAME_TYPE_CHOICES before deciding that wasn't the best way to do things. Anyways, fixing.

**Review by ldko — COMMENTED** [review]
The changes look ok to me, but I have a couple of comments anyway. :)
To me adding another option (that probably should have already existed) to a search filter dropdown doesn't seem like it deserves a minor release--though if @somexpert @runderwood think it does, I will go along with it. Second, before we actually do a release, I would like us to take care of the other open minor issue #111.

**Comment by somexpert** [comment]
I won't argue in favor of a minor release. It really is a very minor addition, and it is really only an _addition_ to the UI, as you could modify the query to search through different name types already. Anyways, I'll go ahead and address the other issue in the next PR.

**Review by ldko — APPROVED** [review]
:+1:

**Comment by runderwood** [comment]
:+1:

**Review by runderwood — APPROVED** [review]

**Pull request merged**